Mathlib/Tactic/Common.lean

```diff
@@ -4,14 +4,12 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
 
-/-
-This file imports all tactics which do not have significant theory imports,
-and hence can be imported very low in the theory import hierarchy,
-thereby making tactics widely available without needing specific imports.
+-- This file imports all tactics which do not have significant theory imports,
+-- and hence can be imported very low in the theory import hierarchy,
+-- thereby making tactics widely available without needing specific imports.
 
-We include some commented out imports here, with an explanation of their theory requirements,
-to save some time for anyone wondering why they are not here.
--/
+-- We include some commented out imports here, with an explanation of their theory requirements,
+-- to save some time for anyone wondering why they are not here.
```

Comment:
What's the reason for this change?

Could you revert changing the block comment to line comments, and maybe switch to `/-!` so that it shows up in the generated docs?

Reply:
Sorry, yes. The linter was complaining and I forgot to revert this after finding a better solution.

Reply:
It can't be `/-!` because it needs to come before the imports for readability.

Reply:
I think it's okay to just put it after the imports, after all.